lib/TWCManager/Vehicle/TeslaMateVehicle.py

```diff
@@ -42,41 +31,19 @@ class TeslaMateVehicle:
         "charger_voltage": ["voltage", lambda a: int(a)],
         "charging_state": ["chargingState", lambda a: str(a)],
     }
-    unknownVehicles = {}
 
     def __init__(self, master):
-        self.__master = master
+        super().__init__(master)
 
-        self.__config = master.config
-        try:
-            self.__configConfig = self.__config["config"]
-        except KeyError:
-            self.__configConfig = {}
-        try:
-            self.__configTeslaMate = self.__config["vehicle"]["TeslaMate"]
-            self.status = self.__config["vehicle"]["TeslaMate"]["enabled"]
-        except KeyError:
-            self.__configTeslaMate = {}
-
-        # Unload if this module is disabled or misconfigured
         if not self.status:
-            self.__master.releaseModule("lib.TWCManager.Vehicle", "TeslaMate")
             return None
 
         # Configure database parameters
-        self.__db_host = self.__configTeslaMate.get("db_host", None)
-        self.__db_name = self.__configTeslaMate.get("db_name", None)
-        self.__db_pass = self.__configTeslaMate.get("db_pass", None)
-        self.__db_user = self.__configTeslaMate.get("db_user", None)
-
-        # Configure MQTT parameters
-        self.__mqtt_host = self.__configTeslaMate.get("mqtt_host", None)
-        self.__mqtt_user = self.__configTeslaMate.get("mqtt_user", None)
-        self.__mqtt_pass = self.__configTeslaMate.get("mqtt_pass", None)
-        self.__mqtt_prefix = self.__configTeslaMate.get("mqtt_prefix", None)
-
-        self.syncTelemetry = self.__configTeslaMate.get("syncTelemetry", False)
-        self.syncTokens = self.__configTeslaMate.get("syncTokens", False)
+        self.__db_host = self.configModule.get("db_host", None)
+        self.__db_name = self.configModule.get("db_name", None)
+        self.__db_pass = self.configModule.get("db_pass", None)
+        self.__db_user = self.configModule.get("db_user", None)
+        self.syncTokens = self.configTeslaMate.get("syncTokens", False)
```

Comment:
```suggestion
        self.syncTokens = self.configModule.get("syncTokens", False)
```

Reply:
Dang, I should have caught that one! I figured it would crash on something more subtle. ;-)